Teacher Workout/Features/Authentication/SignUp/View/SignUpFormView.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+struct SignUpFormView: View {
+    @StateObject var viewModel = SignUpFormViewModel()
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            InputFieldView(label: AppStrings.Authentication.Email.inputLabel.rawValue.localized(),
+                           iconName: "envelope",
+                           placeholder: AppStrings.Authentication.Email.inputPlaceholder.rawValue.localized(),
+                           errorMessage: AppStrings.Authentication.Email.invalidEmailMessage.rawValue.localized(),
+                           fieldData: $viewModel.email,
+                           showError: $viewModel.showEmailError)
+
+            InputFieldView(label: AppStrings.Authentication.Password.inputLabel.rawValue.localized(),
+                           iconName: "lock",
+                           placeholder: AppStrings.Authentication.Password.inputPlaceholder.rawValue.localized(),
+                           isSecureField: true,
+                           fieldData: $viewModel.password,
+                           showError: $viewModel.showPasswordError)
+
+            InputFieldView(label: AppStrings.Authentication.Password.inputConfirmLabel.localized(),
+                           iconName: "lock",
+                           placeholder: AppStrings.Authentication.Password.inputPlaceholder.rawValue.localized(),
+                           isSecureField: true,
+                           fieldData: $viewModel.confirmPassword,
+                           showError: $viewModel.showConfirmPasswordError)
+
+            CheckboxFieldView(isSelected: $viewModel.areTermsAndConditionsAccepted,
```

Comment:
Bug: If we have an error on a field from the top of the Checkbox, the text is not displayed on multiple rows.
![Simulator Screen Shot - iPhone 12 mini - 2021-08-01 at 12 10 59](https://user-images.githubusercontent.com/481425/127765886-229198d1-4765-4a0f-a759-e4233ecf8fbd.png)